pkg/apply/applier_test.go

```diff
@@ -674,22 +674,6 @@ func TestApplier(t *testing.T) {
 					EventType: event.InitType,
 					InitEvent: &testutil.ExpInitEvent{},
 				},
-				{
```

Comment:
More generally, do we have any use-cases where the client cares about the inventory-related tasks? Or could we consider them implementation details and avoid sending events for them?

Reply:
I think we want to send them because they make web requests, which might take up to 30s to timeout if the apiserver is slow. It's usually a good ideal to tell the caller what you're doing when you hand off control like that. It can also be useful when building a progress bar/% (tho we're not doing this yet).

Reply:
TIL the printer don't print those events.... it probably should. 

Plus, it would have helped me find the bug in the tests that was deleting the inventory during a dry-run.